pages/viewBattles.php

```diff
@@ -295,7 +295,8 @@ function viewBattles() {
 
 
         /* View Log */
-        $logs_result;
+        $p1_avatar_size = 200;
+        $p2_avatar_size = 200;
```

Comment:
Assuming this is intentional for new battle UI, since 200 is premium size?

Reply:
Yep just defaults for error handling, the p1 shouldn't be used.